README.md

```diff
@@ -6,23 +6,29 @@ Loop end-to-end automated tests using `detox`
 
 
 ## Build
-
-- `detox build -c ios.sim.debug`
-- `detox build -c ios.sim.release`
+- `npm run build`
 
 ## Test
 
 All tests
 
-`detox test -c ios.sim.debug`
+`npm run all-test`
 
 ### Smoke test
+Accessibility tests that run through the screen elements making sure the are present
+
+- `npm run smoke1-test`
```

Comment:
[nit] I would name all of these targets "test-***". For example, "test-smoke-1", "test-all", etc.